common/src/main/java/com/microsoft/identity/common/internal/migration/AdalMigrationAdapter.java

```diff
@@ -243,10 +242,9 @@ public static boolean loadCloudDiscoveryMetadata() {
                         "Failed to load instance discovery metadata",
                         e
                 );
-                succeeded = false;
             }
         }
 
-        return succeeded;
+        return AzureActiveDirectory.isInitialized();
```

Comment:
There's no visibility guarantee on anything written to sIsInitialized in AzureActiveDirectory...

Reply:
Yuck. Let's get that fixed

Reply:
db7c0e90c7780d55ee8c6e2734bac0ce7b70b1aa